### PR TITLE
Heroku app crash

### PR DESCRIPTION
**Pull request by TravisGM92**
### Participating Group Members:
* Travis McKinstry 
 
### Code Highlights:
   * I think this will fix the bug where the Heroku app wasn't loading. 
 
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Any background context you want to provide?

App kept crashing cause it didn't have access to keys
 
### Issues: 
* Addresses #
* Closes #
 
### Screenshots (if appropriate): 
*
 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed

- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: boneyardproj/settings.py [hunk 1]
```diff
@@ -22,7 +24,7 @@
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = SECRET_KEY
```
Comment (robertorodriguez12) [inline boneyardproj/settings.py:27]: What is this change accomplishing?

File: boneyardproj/settings.py [hunk 2]
```diff
@@ -116,7 +118,7 @@
 
 USE_TZ = True
 
-KEY = config('KEY')
+KEY = KEY
```
Comment (robertorodriguez12) [inline boneyardproj/settings.py:121]: Same with this, Can you explain this to me please?

**Comment by robertorodriguez12** [comment]
Did you add the keys to the heroku config vars?

File: boneyardproj/settings.py (hunk 1 above)
Comment (TravisGM92) [reply boneyardproj/settings.py:27]: 'config' was for using decouple, which I couldn't get to work. Now we are just pulling the environmental key from settings.py file in main_app

**Comment by robertorodriguez12** [comment]
Thank you for the explanations in zoom :)

**Pull request merged**